deploy/cic-spec.yaml

```diff
@@ -987,6 +1074,14 @@ components:
             - CIC_DATA_RECEIVED
             - CIC_AUTH_CODE_ISSUED
             - CIC_ACCESS_TOKEN_ISSUED
+    SessionConfigResponse:
+      required:
+        - "journey_type"
+      type: "object"
+      properties:
+        journey_type:
+          type: "string"
+          example: "face-to-face"
```

Comment:
Do you think we should have an enum for this? If it's going to control frontend routing it's pretty important that it's correct I think